src/containers/notification/components/TwitterShare.tsx

```diff
@@ -29,6 +31,32 @@ import { openTwitterLink } from 'utils/tweet'
 import styles from './TwitterShare.module.css'
 import { getRankSuffix } from './formatText'
 
+const messages = {
+  shareText: 'Share to Twitter',
+  shareTextFans: 'Share With Your Fans',
+  shareTextMilestone: 'Share this Milestone'
+} as const
+
+const TWITTER_NOTIFICATION_TYPE = [
+  NotificationType.ChallengeReward,
+  NotificationType.Milestone,
+  NotificationType.RemixCosign,
+  NotificationType.RemixCreate,
+  NotificationType.TierChange,
+  NotificationType.TrendingTrack
+] as const
+
+const isTwitterNotificationType = (
+  type: any
```

Comment:
░░░░░░░░░░░░░░░░▄▄▄░░░░░░░░░░░░░░░░░
░░░░░░░░░░▄▀▀▀▀▀▀▀▀▀▀▀▀▀▄▄▄░░░░░░░░░
░░░░░░▄▄▀▀░░░░░░░░░░░░░░░░░▀▄▄░░░░░░
░░░░▄█░░░░░░░░░░░░░░░░░░░░░░░░▀▄░░░░
░░░▄▀░░░░░░░░░░░░░░░░░░░░░░░░░░▀▄░░░
░░█▀░░░░░░░░░░░░░░░░░░░░░░░░░░░░▀█░░
░▄█░░░░░░░░░▄▄▄▄▄░░▀▀▀▀░░░░▄░░░░░▀▄░
░█░░░░░░░░▄▄▄▄▄▄░██▀▀▀▀██▄▄▄▄░░░░░▀▄
█▀░░░░░░▀▀██▀▀▀▀██▀░░░░▀██▀▀▀▀█▀░░░█
█░░░░░░░░████░░░░█░░░█░░██░░░██░░░░█
█░░░░░░░▀▀██▄▄▄▄█▀░░░█░░▀██▄▄█▀░░░▄▀
█░░░░░░░░░▀████▀▀▄▄▄▄██▄▄████▀░░░░█░
█░░▄▀░░░░░░░▄▄▄▄██████▀▀██▄▄░░░░░░█░
▀▄▀░░▄▄▄█▀▀▀▀████▀█▀███▀▀▀█████▄▄▄█░
░▀▄░░▀█░░░░░░░░░▀▀▀████▀▀▀░░░░▄▄▀█░░
░░░▀▄░░▀░░░░░░░░░░▀░▀█░░░░░░░░░░█░░░
░░░░░▀▄░░░░░░░░░░░░░░▀░░░░░░░░▄▀░░░░
░░░░░░░▀▀▄▄░░░░░░░░░░░░░░░░▄▄▀░░░░░░
░░░░░░░░░░░▀▀█▄▄▄▄▄▄▄▄▄▄▄█▀▀░░░░░░░░

Reply:
does it not work to just put
`type: TWITTER_NOTIFICATION_TYPE`
?